### ISSUE TITLE
videos is black

### ISSUE THREAD

**Issue by geeve-research** [kohya-ss/musubi-tuner#5]
INFO:hunyuan_model.vae:Loading 3D VAE model (884-16c-hy) from: /root/musubi-tuner/HunyuanVideo/ckpts/hunyuan-video-t2v-720p/vae/pytorch_model.pt
INFO:hunyuan_model.vae:VAE to dtype: torch.float16
INFO:__main__:Set chunk_size to 32 for CausalConv3d
/root/musubi-tuner/hv_generate_video.py:63: RuntimeWarning: invalid value encountered in cast
  x = (x * 255).numpy().astype(np.uint8)
INFO:__main__:Sample save to: /root/output/20250103-060011_1234.mp4

how do i fix this? inference script has issues

**Comment by geeve-research** [comment]
@kohya-ss

**Comment by Bocchi-Chan2023** [comment]
disable overclocking.
Show used commands first

**Comment by geeve-research** [comment]
@Bocchi-Chan2023 @kohya-ss i used root@modal:~/musubi-tuner# python hv_generate_video.py --video_size 544 960 --video_length 5 --infer_steps 30 --prompt "ct, apple on a tree." --save_path /root/output --output_type both --dit /root/musubi-tuner/HunyuanVideo/ckpts/hunyuan-video-t2v-720p/transformers/mp_rank_00
_model_states.pt --attn_mode sdpa --vae /root/musubi-tuner/HunyuanVideo/ckpts/hunyuan-video-t2v-720p/vae/pytorch_model.pt --vae_chunk_size 32 --vae_spatial_tile_sample_min_
size 128 --text_encoder1 /root/musubi-tuner/HunyuanVideo/ckpts/text_encoder --text_encoder2 /root/musubi-tuner/HunyuanVideo/ckpts/text_encoder_2 --seed 1234 --lora_multipli
er 1.0 --lora_weight /root/output/ct-lora.safetensors

**Comment by geeve-research** [comment]
when i add --vae_dtype bf16 at the end  i get error too hv_generate_video.py: error: unrecognized arguments: --vae_dtype bf16

**Comment by Bocchi-Chan2023** [comment]
please tell me your environment. Did you oc/uv?

**Comment by geeve-research** [comment]
i'm connected to a modal labs h100 gpu server. inside vscode, using vscode terminal. @Bocchi-Chan2023 @kohya-ss

**Comment by nivibilla** [comment]
@kohya-ss I am also getting black videos with lora inferencing. I suspect it might be due to the recent changes made. I trained my model yesterday (9th Jan) at commit fd70762, but doing inference at main causes black video. Since there were changes to the attention file and other model files do i need to retrain the model? For sanity i checked by reverting back to that commit and it works fine.

**Comment by nivibilla** [comment]
> Could you try reducing the learning rate to about half or 1/5 and re-training?
i reran the same command with no changes apart from the learning rate and updated git code. I am now going OOM on a 48gb gpu which i wasnt before at batch size 4 with 8bit model and 8bit adam

**Comment by nivibilla** [comment]
@kohya-ss  Getting black videos even after retraining. Also getting black video even without lora and straight base model inference. Not sure whats happening.
